content/en/beginner-install.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Beginner Installation Guide
+sidebar: false
+---
+
+{{< admonition tip >}}
+This is the beginner installation guide.
+If you are comfortable with using a terminal and happy to learn
+how to use a package manager, check out
+[the main installation guide](./install.md)!
+{{< /admonition >}}
+
+- [JupyterLite](#jupyterlite)
+- [Scientific Python Distributions](#distributions)
+- [Installing globally with `pip`](#pip-global)
+
+<a name="jupyterlite"></a>
+
+## JupyterLite
+
+To try out SciPy, you don't even need to install it!
+You can use SciPy in your browser at https://jupyter.org/try-jupyter/lab/ -
+just write `import scipy` in one of the notebook "cells" and hit play.
+
+For next steps, see [the SciPy user guide][scipy-user-guide].
+
+[scipy-user-guide]: https://docs.scipy.org/doc/scipy/tutorial/
+
+<a name="distributions"></a>
+
+## Scientific Python Distributions
+
+Python distributions provide the language itself, along with the most
+commonly used packages and tools. These downloadable files require
+little configuration, work on almost all setups, and provide all the
+most commonly used scientific Python tools.
+[Anaconda](https://www.anaconda.com/download/) works on Windows, Mac,
+and Linux, and is best suited to beginning users.
+Other options include:
+
+- [WinPython](https://winpython.github.io): Another free distribution
+  including scientific packages and the Spyder IDE; Windows only, but
+  more actively maintained and supports the latest Python 3 versions.
```

Comment:
More actively maintained than Anaconda?

Reply:
@CAM-Gerlach thought I'd ping in case you have any opinions on the Python Distributions section here! Hasn't been updated in a while...